doc/thing/samples/sensor-samples/pwm.rst

```diff
@@ -0,0 +1,27 @@
+pwm
+===
+
+Overview
+--------
+
+The pwm example blinks a led using pwm. The purpose of this is sample is to
+show how to use pwm in Zephyr.
+
+-------------------------------------------------------------------------------
+
+Requirements
+------------
+
+The demo assumes that the positive terminal of a led is connected to the pin
+specified down below:
+
+- 0.03 pin (for KNoT Mesh board)
```

Comment:
The KNoT Mesh board is not on the documentation 'supported boards' section. This way I would not add this information for now.

Reply:
Done